app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -8,7 +8,7 @@ module ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcherMixin
     'Node'                  => %w(NodeReady NodeNotReady Rebooted NodeSchedulable NodeNotSchedulable InvalidDiskCapacity
                                   FailedMount),
     'Pod'                   => %w(Scheduled FailedScheduling FailedValidation HostPortConflict DeadlineExceeded
-                                  FailedSync OutOfDisk NodeSelectorMismatching InsufficientFreeCPU
```

Comment:
@simon3z I didn't realize `containergroup_failedsync` was an `MiqEvent` not an `EmsEvent, my mistake